src/Atoms/InfoLabel/InfoLabel.js

```diff
@@ -1,55 +1,41 @@
 import PropTypes from 'prop-types';
 import InfoLabelStyled from './InfoLabelStyled';
+import { TIME } from '../../Consts/infoLabelConsts';
 
-const InfoLabel = ({ classInfo, dogInfo, dateOfContest = '' }) => {
+const InfoLabel = ({
+  classInfo,
+  dogInfo,
+  startDateOfContest = '',
+  endDateOfContest = '',
+}) => {
   const { dogsAmount, isCompleted } = classInfo || [];
   const { exercisesCompleted, exercisesAmount } = dogInfo || [];
 
   const exercisesComplete =
     exercisesAmount !== undefined && exercisesCompleted === exercisesAmount;
 
   const todayDate = new Date();
-  let date;
   let dateTextInfo;
 
-  if (dateOfContest) {
-    date = dateOfContest.toISOString().substring(0, 10);
-  } else {
-    date = '';
-  }
-
-  Date.prototype.addHours = function (h) {
-    this.setTime(this.getTime() + h * 60 * 60 * 1000);
-    return this;
-  };
-  Date.prototype.subtractMinutes = function (min) {
-    this.setTime(this.getTime() - min * 60 * 1000);
-    return this;
-  };
-
-  if (!dateOfContest) {
+  if (!startDateOfContest) {
     dateTextInfo = '';
-  } else if (dateOfContest < todayDate.subtractMinutes(15)) {
-    dateTextInfo = 'in-past';
-  } else if (
-    todayDate > dateOfContest.subtractMinutes(15) &&
-    todayDate < dateOfContest.addHours(5)
-  ) {
-    dateTextInfo = 'in-progress';
-  } else if (dateOfContest > todayDate) {
-    dateTextInfo = 'in-future';
+  } else if (endDateOfContest < todayDate) {
+    dateTextInfo = TIME.PAST;
+  } else if (todayDate >= startDateOfContest && todayDate <= endDateOfContest) {
+    dateTextInfo = TIME.PRESENT;
+  } else if (startDateOfContest > todayDate) {
+    dateTextInfo = TIME.FUTURE;
   }
 
   return (
     <InfoLabelStyled
       isClassCompleted={isCompleted}
       areExercisesCompleted={exercisesComplete}
       dateTextInfo={dateTextInfo}
+      dogsAmount={dogsAmount}
```

Comment:
instead of all these specific props, you could give them one that would let the styled component which style to choose, like: `color={areExecisesComplete? "green" : "white"}`